doctrine/doctrine-bundle/1.6/config/packages/doctrine.yaml

```diff
@@ -1,7 +1,15 @@
+parameters:
+    # adds a fallback URL if the env var is not set
+    # this is useful to allow you to run cache:warmup
+    # even when your environment variables don't yet exist
+    # you should not need to change this value
+    env(DATABASE_URL): '%doctrine.database_server%://'
```

Comment:
Is it possible to specify a default value for the `DATABASE_URL` environment value in some other way (i.e. not below `parameters`)? I *could* move this to `services.yaml` as @symfony-flex-server suggests, but it's really better here, as the user should not really configure this.